mobile/src/test/java/org/openhab/habdroid/util/UtilTest.java

```diff
@@ -42,16 +44,31 @@ public void normalizeUrl() throws Exception {
 
     @Test
     public void parseSitemapList() throws Exception {
-        List<OpenHABSitemap> sitemapList = Util.parseSitemapList(createSitemapDocument());
+        // openHAB 1
+        List<OpenHABSitemap> sitemapList = Util.parseSitemapList(createSitemapOH1Document());
         assertFalse(sitemapList.isEmpty());
 
         // Should be sorted, null first
```

Comment:
Please move this comment to a message of the assertEquals and explain why the first label should be empty, please :)

Reply:
I wait for #623 as sitemap labels arent null anymore.